bindings/python/tests/test_trans.py

```diff
@@ -43,10 +43,10 @@ def test_transsymbstates():
 
 def test_transition_operations(prepare_automaton_a):
     nfa = mata.Nfa(10)
-    nfa.add_trans_raw(3, ord('c'), 4)
+    nfa.add_transition(3, ord('c'), 4)
     assert nfa.has_trans_raw(3, ord('c'), 4)
     trans = mata.Trans(4, ord('c'), 5)
-    nfa.add_trans(trans)
+    nfa.add_transition_object(trans)
```

Comment:
If I am not mistaken, we have decided in the meeting that we will remove this overload of former `add_trans()`, now `add_transition()` completely, so the following might be no longer relevant, but I am not necessarily happy with the use of `object`. That being said, I cannot come up with a better name anyway. We can keep `add_transition_object()`.

Reply:
I have no issue with removing this function. Will do it tommorow. I'm satisfied with the single one.

Reply:
I rethough this, and will fix it after Martin's cleanup. I skimmed through the bindings, and there are more of these overloaded versions, that should be removed.